bookmark.h

```diff
@@ -0,0 +1,56 @@
+#ifndef BOOKMARK_H
+#define BOOKMARK_H
+#include <string>
+#include <iostream>
+#include <conio.h>
+
+using namespace std;
+class Bookmark {
+        public:
+            void displayMainMenu();
```

Comment:
display main menu should not be part of the bookmark class. Classes should be dump as possible. Here in your case, the Bookmark class knows about displaying the main menu. classes are equal as objects in real life. Do bookmarks in real life know about the main menu?